BESMod/Systems/Hydraulical/BaseClasses/PartialHydraulicSystem.mo

```diff
@@ -124,7 +124,8 @@ partial model PartialHydraulicSystem
       Placement(transformation(extent={{-12,-104},{90,28}})));
 
   replaceable BESMod.Systems.Hydraulical.Transfer.BaseClasses.PartialTransfer transfer(
-      dp_nominal=fill(0, transfer.nParallelDem))                                                                                                                                                                                                         constrainedby
+      dp_nominal=fill(0, transfer.nParallelDem), TTran_nominal=
```

Comment:
`Tran` should be `Tra`, always three letters.

Reply:
I changed the naming, thanks!